plottr/gui/data_display.py

```diff
@@ -32,14 +32,13 @@ def __init__(self, parent: Optional[QtWidgets.QWidget] = None,
 
     def _makeItem(self, name: str) -> QtWidgets.QTreeWidgetItem:
         shape = self._dataShapes.get(name, tuple())
-        label = f"{name} [{self._dataStructure.label(name)}]"
-        deps = "("
+        label = f"{self._dataStructure.label(name)}"
+        deps = ""
         for i, d in enumerate(self._dataStructure.axes(name)):
             if i > 0:
                 deps += ", "
             axlabel = self._dataStructure.label(d)
-            deps += f"{d} [{axlabel}]"
-        deps += ")"
+            deps += f"{axlabel}"
```

Comment:
Personally I find it cleaner to write this with a list compreshension and and join.

```python
        axlabels = [self._dataStructure.label(d) for d in
                    self._dataStructure.axes(name)]
        deps = ",".join(axlabels)
```

Reply:
agreed -- i changed that.

Reply:
hmmm, now there's a mypy error that seems to come from the list comprehension. how do you fix that without reverting to the previous verbose code?

Reply:
So the problem is that label many return `None` so axlabel is a list of optional str. I have fixed that by just converting any Nones to str